core/trino-main/src/main/java/io/trino/sql/planner/LogicalPlanner.java

```diff
@@ -598,6 +629,14 @@ private RelationPlan getInsertPlan(
                 statisticsMetadata);
     }
 
+    private List<Symbol> getVisibleFieldMappings(RelationPlan plan)
+    {
+        return IntStream.range(0, plan.getFieldMappings().size())
```

Comment:
@martint Is this a critical code path or can we use Stream here ?

Reply:
No, it's fine.